### PR TITLE
Update WEC-Sim Applications CI to reference the main branch

### PR DESCRIPTION
**Pull request by akeeste**
This PR updates the CI workflows for the Applications repo and any documentation references within it to reference the main branches WEC-Sim/WEC-Sim#1215

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # WEC-Sim_Applications
 
-[![master build status](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-master.yml/badge.svg)](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-master.yml)
+[![main build status](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-main.yml/badge.svg)](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-main.yml)
```
Comment (H0R5E) [inline README.md:3]: @akeeste, you either need to rename the workflow file to `run-tests-main.yml` or leave the links to the files unaltered. Same goes in the Applications directory.

**Comment by H0R5E** [comment]
I assume you're going to push this to `main`, yes?

**Comment by akeeste** [comment]
Yes before merging I'll make the main branch and make this PR go into main

**Comment by akeeste** [comment]
Thanks @H0R5E! I reran the tests here and the documentation tests on WEC-Sim/WEC-Sim#1235. Everything seems to be passing now. 

The last thing before merging #59 and WEC-Sim/WEC-Sim#1235, is to update the default branch on the Applications repo to main. If you can rename master directly to main, it looks like we'll have the added benefit of having API or web requests to master automatically redirect to the main branch https://github.com/github/renaming?tab=readme-ov-file#renaming-existing-branches

**Comment by akeeste** [comment]
I'm not sure how renaming 'master' to 'main' will go if the 'main' branch already exists. If we need to take a couple extra steps to get the API / web requests automatically redirecting, it's worth it; e.g. 
1. update this PR back into master so that it is not closed
2. delete the current main branch
3. then rename master to main with auto redirecting

**Comment by H0R5E** [comment]
@akeeste, I'm on it.

**Comment by H0R5E** [comment]
@akeeste, I found another reference to the master branch in the [Wave_Markers folder](https://github.com/WEC-Sim/WEC-Sim_Applications/blob/aeb951c796a65eae5c42558cbf0d208f89ee5588/Wave_Markers/README.md?plain=1#L10). Can you fix this one, also?

**Comment by akeeste** [comment]
> @akeeste, I found another reference two references to the master branch in the [Wave_Markers folder](https://github.com/WEC-Sim/WEC-Sim_Applications/blob/aeb951c796a65eae5c42558cbf0d208f89ee5588/Wave_Markers/README.md?plain=1#L10). Can you fix this one, also?

Good catch, done! I did one more search across all Applications repo files but did not find any other references. let me know if you catch anything else.

**Comment by akeeste** [comment]
I also just confirmed that this repo's Mooring test does not checkout a specific master or main branch from the WEC-Sim/MoorDyn repo. There's only one branch and the default is always used for the test, so that repo should not affect this PR

**Comment by H0R5E** [comment]
@akeeste, I'm going to merge this now. Thanks for sorting this out and the tip about renaming the branches.

**Pull request merged**